pkg/util/runtime/features.go

```diff
@@ -139,9 +139,9 @@ var (
 		FeatureRollingUpdateFix:                 false,
```

Comment:
I realize this isn't part of your PR, but could you please move `FeatureRollingUpdateFix` so that it's in alphabetical order?

Reply:
Sounds good!